### PR TITLE
[Experimental] prefetch in execute_block

### PR DESCRIPTION
**Pull request by gelash**

### CODE REVIEW

File: aptos-move/aptos-vm/src/block_executor/mod.rs [hunk 1]
```diff
@@ -133,6 +145,37 @@ impl BlockAptosVM {
...
+            })
+            .collect();
+
+        IO_POOL.install(|| {
+            addrs.par_iter().for_each(|addr| {
+                let ap_coin = AccessPath::resource_access_path(ResourceKey::new(
+                    *addr,
+                    CoinStoreResource::struct_tag(),
+                ));
+                let ap_seq = AccessPath::resource_access_path(ResourceKey::new(
```
Comment (georgemitenkov) [inline aptos-move/aptos-vm/src/block_executor/mod.rs:165]: @gelash Why do you access CoinStore twice here?

Comment (gelash) [reply aptos-move/aptos-vm/src/block_executor/mod.rs:165]: ouch, thanks!

File: aptos-move/aptos-vm/src/block_executor/mod.rs [hunk 2]
```diff
@@ -133,6 +145,37 @@ impl BlockAptosVM {
...
+            .flat_map(|txn| {
+                if let Transaction::UserTransaction(t) = txn {
+                    Some(t.sender())
+                } else {
+                    None
+                }
+            })
+            .collect();
+
+        IO_POOL.install(|| {
```
Comment (msmouse) [inline aptos-move/aptos-vm/src/block_executor/mod.rs:159]: why not `spawn`? -- you don't need to wait for all the results to come back, right?

**Comment by davidiw** [comment]
Going to close these out since this effort is becoming more focused and these are sprinkled throughout.

Maybe we can add a tag to PRs so we can easily filter them out?

**Pull request closed without merging**